### PR TITLE
JSHint code cleanup

### PR DESCRIPTION
**Pull request by PierreBesson**
Fixing #2506 

I cleaned up all the little issues I could find.
One change I did on webapp/app/components/alert/_alert.service.js is not really satisfying. I just skip the warning for now, the file should be refactored later.

### CODE REVIEW

**Comment by PierreBesson** [comment]
As I expected, I found out that there are more jshint issues if I answer differently to the questions. So this cleanup could take a while...

**Comment by PierreBesson** [comment]
I think the cleanup is done now but I might have overlooked some configuration possibilities.

File: app/templates/gulpfile.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 'use strict';
 
 var gulp = require('gulp'),
-    gutil = require('gulp-util'),
+//    gutil = require('gulp-util'),
```
Comment (deepu105) [inline app/templates/gulpfile.js]: Why is this commented? If it is not ysed we can remove no need to keep it commented

**Comment by deepu105** [comment]
ill remove the commented line

**Pull request merged**